blas install documentation for Ubuntu

On ubuntu 16.10 I get this error:

```
(venv3) luis@spinoza:~/lab/sandbox/ble/pb_pythran$ python setup.py bdist_wheel --enable-pythran
/usr/bin/ld : ne peut trouver -lcblas
collect2: error: ld returned 1 exit status
/usr/bin/ldÂ : ne peut trouver -lcblas
collect2: error: ld returned 1 exit status
running bdist_wheel
```

--- a/README.rst
+++ b/README.rst
@@ -38,7 +38,7 @@ Using ``pip``
 
    Pythran depends on a few Python modules and several C++ libraries. On a debian-like platform, run::
 
-        $> sudo apt-get install libgmp-dev libblas-dev
+        $> sudo apt-get install libgmp-dev libatlas-base-dev
         $> sudo apt-get install python-dev python-ply python-networkx python-numpy
 
 2. Use ``easy_install`` or ``pip``::

